AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/StorageEngineAdapter+SQLite.swift

```diff
@@ -20,12 +20,16 @@ final class SQLiteStorageEngineAdapter: StorageEngineAdapter {
         guard let documentsPath = getDocumentPath() else {
             preconditionFailure("Could not create the database. The `.documentDirectory` is invalid")
         }
-        let dbFilePath = documentsPath.appendingPathComponent("\(databaseName).db")
+        var dbFilePath = documentsPath.appendingPathComponent("\(databaseName).db")
         let path = dbFilePath.absoluteString
 
         let connection: Connection
         do {
             connection = try Connection(path)
+
+            var setValues = URLResourceValues()
```

Comment:
Kind of a weird name--it sounds like it wants to be a setter method.

Reply:
Hmm. thanks, changed to:
`urlResourceValues`